src/tz_windows.rs

```diff
@@ -27,43 +32,35 @@ const WINDOWS_GLOBALIZATION_CALENDAR: &[wchar_t] = &wstring!(
     0
 );
 
-static INITALIZED: sync::Once = sync::Once::new();
-static mut FACTORY: Result<Unknown<IActivationFactory>, HRESULT> = Err(0);
+static mut FACTORY: OnceCell<Result<Unknown<IActivationFactory>, HRESULT>> = OnceCell::new();
```

Comment:
this doesn't need to be a `static mut` anymore I believe since `OnceCell` gives interior mutability.

Reply:
core (no_std) has OnceCell now.. but it's still not stable.. maybe feature gate it ?
https://doc.rust-lang.org/nightly/core/cell/struct.OnceCell.html